src/templates/partials/redis.hbs

```diff
@@ -0,0 +1,31 @@
+# {{output.header}}
+# {{{output.link}}}
+
+# Running Redis with TLS requires you to build Redis specifically for TLS
+# This can be done when making redis using "make && make BUILD_TLS=yes"
+
+################################# TLS/SSL #####################################
+port 0
+tls-port 6379
+tls-cert-file redis.crt
+tls-key-file redis.key
+tls-ca-cert-file ca.crt
+tls-ca-cert-dir /etc/ssl/certs
+tls-auth-clients yes
+tls-replication yes
+tls-cluster yes
+{{#unless (includes "old" form.config)}}
+# Running Redis with TLS 1.3 requires OpenSSL 1.1.1 or greater
+{{/unless}}
+{{#if (includes "intermediate" form.config)}}
+tls-protocols "TLSv1.2 TLSv1.3"
```

Comment:
All of these should be programatically generated, there should in general not be any manual setting based on things. This is so all the profiles update once the JSON file is updated.

Reply:
output.protocols instead should be used. Got it.